plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1226,7 +1227,7 @@ private boolean checkOvsNetwork(final String networkName) {
             return true;
         }
 
-        final Script command = new Script("/bin/sh", _timeout);
+        final Script command = new Script("/bin/sh", _timeout.getMillis());
```

Comment:
Please consider adding an overridden version of the `Script(String, int)` constructor that accepts `Duration` to encapsulate this type conversion.